packages/flutter/test/rendering/paragraph_test.dart

```diff
@@ -270,14 +270,17 @@ void main() {
     // rendering widths in tests.
     // TODO(#12357): Figure out why this is, and fix it (if needed) once Blink
     // text rendering is replaced.
+    // anyOf for heights is needed because libtxt and Blink calculate selection
+    // rectangles differently.
+    // TODO: remove this when Blink is replaced.
     expect(boxes[0].toRect().width, anyOf(14.0, 13.0));
-    expect(boxes[0].toRect().height, 13.0);
+    expect(boxes[0].toRect().height, anyOf(13.0, 26.0));
```

Comment:
this seems wrong to me. Why would 26 be the right answer?
This just seems like a bug in libtxt.

Reply:
libtxt doesn't track the height of each individual glyph in a line.

The rectangle returned by libtxt's version of getBoxesForRange will have a height matching the height of the entire line (similar to the rectangle typically used when highlighting a selection in a browser or text editor)

Reply:
It's not each glyph per se, it's each text span (notice how "b" and "c" in this test have the same box). We're going to need to track that so that we can draw backgrounds behind each span of text.

It presumably does track the height of each text span since the height of each text span is just the fontSize of that span. The bottom of the rectangle is baseline+descent, the top is baseline-ascent.